charts/sysdig/templates/configmap.yaml

```diff
@@ -21,6 +21,10 @@ data:
       enabled: true
 {{- end }}
 {{- end }}
+{{- if eq .Values.sysdig.disableCaptures true }}
+    {{- if hasKey .Values.sysdig.settings "sysdig_capture_enabled" }}{{ fail "sysdig_capture_enabled options in both .sysdig.disableCaptures and sysdig.settings.collesysdig_capture_enabledctor"}}{{ end }}
```

Comment:
We are having this pattern in several places, and perhaps for next time we should think about extracting a helper function to  automate this.

Reply:
Agree, as I already sent to PR to IBM too, I'll schedule that change for 1.7.19